client/client-multi/client-c/src/kaa/kaa_user.c

```diff
@@ -153,6 +171,8 @@ kaa_error_t kaa_user_manager_create(kaa_user_manager_t **user_manager_p
 void kaa_user_manager_destroy(kaa_user_manager_t *self)
 {
     if (self) {
+        kaa_list_destroy(self->attach_endpoints, NULL);
```

Comment:
There is a memory leak. Need to deallocate memory used by the endpoint_token field of an endpoint_info_t structure.